src/js/descript.js

```diff
@@ -135,5 +167,41 @@
         return containerName ? this._containers[containerName] : this._containers;
     };
 
+    /**
+     * Replaces string patterns in inline scripts. Patterns can be
+     * @param containerName
```

Comment:
Correct me if I'm wrong, but isn't the script not removed from the default container?